features/show_shf_application.feature

```diff
@@ -0,0 +1,177 @@
+
+  @user
+  Scenario: Listing incoming Applications restricted for Non-admins
+    Given I am logged in as "hans_waits@waiting.se"
+    And I am on the "membership applications" page
+    Then I should see t("errors.not_permitted")
+
+
+  @admin
+  Scenario: Clicking the edit button on show page
+    Given I am logged in as "admin@shf.se"
+    When I am on the "application" page for "nils_member@bowwowwow.se"
+    Then I should see t("activerecord.attributes.shf_application.state/accepted")
+    And I click on t("shf_applications.edit_shf_application")
+    Then I should be on the "edit application" page for "nils_member@bowwowwow.se"
+
+  @user
+  Scenario: User does not see edit-link
+    Given I am logged in as "emma@random.com"
+    When I am on the "Landing" page
+    Then I should see t("menus.nav.users.apply_for_membership")
+    And I should not see t("menus.nav.users.my_application")
+
+
+  @admin
+  Scenario: Admin sees business categories for user under_review
+    Given I am logged in as "admin@shf.se"
+    When I am on the "application" page for "emma_under_review@happymutts.se"
+    Then I should see "rehab"
+
+  @admin
+  Scenario: Admin sees business categories for user that is ready_for_review
+    Given I am logged in as "admin@shf.se"
+    When I am on the "application" page for "hans_ready_for_review@happymutts.se"
+    Then I should see "dog grooming"
+
+  @admin
+  Scenario: Admin sees business categories for user that is waiting_for_applicant
+    Given I am logged in as "admin@shf.se"
+    When I am on the "application" page for "emma_waits@waiting.se"
+    Then I should see "Psychologist"
+
+
+  @admin
+  Scenario: Admin sees business categories for user that is accepted
+    Given I am logged in as "admin@shf.se"
+    When I am on the "application" page for "nils_member@bowwowwow.se"
+    Then I should see "Groomer"
+
+
+  @selenium, @user
+  Scenario: User sees the status translated correctly (default locale, then :en, then :sv)
+    Given I am logged in as "hans_waits@waiting.se"
+    And I set the locale to "sv"
+    When I am on the "application" page for "hans_waits@waiting.se"
+    Then I should see t("activerecord.attributes.shf_application.state/waiting_for_applicant")
+    When I click on "change-lang-to-english"
+    And I wait for 1 seconds
+    Then I should see t("activerecord.attributes.shf_application.state/waiting_for_applicant", locale: :en)
+    And I should not see t("activerecord.attributes.shf_application.state/waiting_for_applicant", locale: :sv)
+    When I click on "change-lang-to-svenska"
+    And I wait for 1 seconds
+    Then I should see t("activerecord.attributes.shf_application.state/waiting_for_applicant", locale: :sv)
+    And I should not see t("activerecord.attributes.shf_application.state/waiting_for_applicant", locale: :en)
+
+
+  @selenium, @member
+  Scenario: Member sees the status translated correctly (default locale, then :en, then :sv)
```

Comment:
Is this meant to be empty?  Or should be removed?

Reply:
whoops.  No - shouldn't be empty.